languages/elixir/exercises/concept/exceptions/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+## General
+
+- Read the documentation:
+  - [Elixir Getting Started - Errors][getting-started-errors]
+  - [Elixir documentation - defexception][defexception]
+  - [Elixir documentation - Exception Behaviour][exception-behaviour]
+- Read the code snippets from the introduction
+
+## 1. Error for Division by Zero
+
+- implement the module, specifying the message using `defexception`
+
+## 2. Error when encountering stack underflow
+
+- implement the module, specifying the message using `defexception`
+- also implement `exception/1`
```

Comment:
"you can use [one of the _Exception Behaviour_ callbacks](https://hexdocs.pm/elixir/Exception.html#c:exception/1) to define an exception whose message changes based on the arguments passed to `raise/2`" or something like that? My goal here is to avoid calling `exception/1` by its name in the text, but also to give context why the student needs that callback.

Reply:
```suggestion
- You can use one of the Exception Behaviour callbacks to define an exception whose message changes based on the arguments passed to `raise/2`.
```

Reply:
That seems reasonable to me :slightly_smiling_face: